ecosystem/indexer-grpc/indexer-grpc-utils/src/config.rs

```diff
@@ -8,6 +8,7 @@ use std::path::PathBuf;
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub struct GcsFileStore {
     pub gcs_file_store_bucket_name: String,
+    pub gcs_file_store_bucket_sub_dir: Option<String>,
```

Comment:
Probably best to use PathBuf, unless you want to assert that this is a single dir name and not `a/path/like/this`. It'll also help deal with leading / trailing slashes.